### DISCUSSION TITLE
[WARN] too many colons in address

### DISCUSSION THREAD

**Discussion by 4rt3mT** [xjasonlyu/tun2socks#132] [General]
When i ping its okay: 
![Без имени](https://user-images.githubusercontent.com/91070756/169107149-76dd0f26-a06c-407b-91e2-f88af0aa8d37.png)

But when i try download something it says:

WARN[0574] [TCP] dial 93.93.128.193:80: connect to socks5:*my proxy* dial tcp: address socks5:*my proxy* too many colons in address

**Comment by xjasonlyu** [comment]
it looks like it’s your address issue. what’s exact your cli parameters?

**Reply by 4rt3mT** [reply]
Where i can find them?

I said "download something" i mean "sudo apt-get install *something*" dont works, but ping successfull

**Reply by xjasonlyu** [reply]
yeah, I mean, what’s your command line parameters?

**Reply by 4rt3mT** [reply]
no parameters dude
ssh connection from windows then just apt-get in terminal

**Reply by 4rt3mT** [reply]
Ohh, i understand sorry, wait a second

**Reply by 4rt3mT** [reply]
'tun2socks -device tun0 -proxy socks5:*my_proxy* -interface eth0'

u mean this?

**Reply by 4rt3mT** [reply]
I think i understand what is going on 

I forgot // in -proxy


but different error 

` connect: no route to host`

![Без имени](https://user-images.githubusercontent.com/91070756/169153178-ee340840-a3f8-41a9-aa06-29cf4732b378.png)

**Reply by 4rt3mT** [reply]
Just done all from scratch and it works.

conclusion: crooked hands
